lighthouse-core/gather/connections/connection.js

```diff
@@ -108,6 +129,11 @@ class Connection {
         return object.result;
       }));
     }
+
+    if (this._recordMessages) {
```

Comment:
yes, figured that would be distracting and not make very much sense without the request payload